api/src/test/java/org/openmrs/api/CohortServiceTest.java

```diff
@@ -499,7 +503,7 @@ public void removePatientFromCohort_shouldNotFailIfCohortDoesNotContainPatient()
 		// make a patient
 		Patient patientToAddThenRemove = new Patient(4);
 		// verify that the patient is not already in the Cohort
-		assertFalse(service.getCohort(2).contains(patientToAddThenRemove));
+		assertFalse(service.getCohort(2).contains(patientToAddThenRemove.getPatientId()));
```

Comment:
I would imagine that you have a new contains method that takes in patient object that internally delegates to teh old one so you wouldn't need these changes